browser/BUILD.gn

```diff
@@ -12,6 +12,10 @@ import("//extensions/buildflags/buildflags.gni")
 import("//third_party/widevine/cdm/widevine.gni")
 import("//third_party/protobuf/proto_library.gni")
 
+declare_args() {
```

Comment:
I would also move the brave_safetynet_api_key config stuff to components/brave_rewards re https://github.com/brave/brave-core/pull/3731/files#r336701998 layering issue

Reply:
done